Path-Check/covid-safe-paths
# Fix: Add trusted source button overlaps "paste url" text box

#### Description:
Fixes #674 

Notes: Ideally we should get rid of the "Add trusted source" button while adding a new url. While adding a new url the user shouldn't be given options to add more trusted sources. Right now the user can click that button and it presents him with another drop down.

#### Updates:
1. Added a keyboard avoiding view in commons. This new component can be used around other text boxes which potentially may have the same issue.
2. Cleaned up repeated code.

#### How to test:
1. Get onboarded
2. Dashboard -> Choose health auth -> Add auth via URL
3. Click on textbox to enter/paste url
4. Add multiple to see if the view scrolls.
![Screen Shot 2020-05-13 at 11 54 33 PM](https://user-images.githubusercontent.com/24991867/81891390-4684a880-9576-11ea-96a6-a8b0d4680875.png)
